### ISSUE TITLE
Crashed

### ISSUE THREAD

**Issue by fruity4pie** [avocode/avocode#1599]
IMPORTANT: Please try to describe how did the crash happen.

App v. `2.14.8`

When I had worked in laptop on UBUNTU, Avocode had opened in second monitor, after 30-40min it had crashed. Just crashed on my system. I don't why it's happened. Maybe it have memory leaks ?

**Comment by vojtatranta** [comment]
This happens regularly, am I understand it correctly?

**Comment by fruity4pie** [comment]
No, not regulary. When Im using 2 monitors and large amount of memory.

31 авг. 2017 г. 19:00 пользователь "Vojtěch Tranta" <
notifications@github.com> написал:
